src/tools/genpolicy/src/mount_and_storage.rs

```diff
@@ -113,7 +115,9 @@ pub fn get_mount_and_storage(
             false
         };
         get_empty_dir_mount_and_storage(settings, p_mounts, storages, yaml_mount, memory_medium);
-    } else if yaml_volume.persistentVolumeClaim.is_some() || yaml_volume.azureFile.is_some() {
+    } else if yaml_volume.persistentVolumeClaim.is_some() {
+        get_persistent_volume_claim_mount(settings, yaml_mount, yaml_volume, p_mounts, storages, persistent_volume_claims, "rprivate", "rw");
```

Comment:
We might need to support "ro" access for pvc based on https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/readonlymany-disks#:~:text=You%20can%20now%20reference%20the%20new%20PersistentVolumeClaim%20in,mode%20to%20multiple%20nodes%20at%20the%20same%20time.

If so, may need to do something similar to https://github.com/microsoft/kata-containers/pull/175/files#diff-bfe7240cc8e3920d3200b19d567341adcfb828a69b91376faa7f6d48342b8295

If not, we may be able to remove `access` parameter from  `get_persistent_volume_claim_mount`, since it's always "rw"